assets/dev/js/editor/editor.js

```diff
@@ -13,8 +13,43 @@ export class Editor extends EditorBase {
 
 		super.onPreviewLoaded();
 	}
+
+	/**
+	 * Function loadModules().
+	 *
+	 * Dynamically inject modules.
+	 * Some of the modules can be injected as promise of their import
+	 *
+	 * @returns {Promise<void>}
+	 */
+	async loadModules() {
```

Comment:
Currently what i did is that the backend(php) load a `module.js` which `import()` being injected to `elementor.modules`, when it reach editor loading, it will be waiting for all injected imports to be loaded first, then it will continue the original load.

@rotemee, do you have better idea?